pkg/activator/stat_reporter.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package activator
+
+import (
+	"github.com/gorilla/websocket"
+	"go.uber.org/zap"
+	"knative.dev/serving/pkg/autoscaler/metrics"
+)
+
+// RawSender sends raw byte array messages with a message type
+// (implemented by gorilla/websocket.Socket).
+type RawSender interface {
```

Comment:
Could we just define a function var instead of an interface?

Reply:
but then we couldn't pass the DurableSendingConnection in from activator/main.go?

Reply:
We can pass DurableSendingConnection.SendRaw?

Reply:
Hm. What would be the advantage of that? This way seems a bit more idiomatic to me?